docker-compose.yaml

```
@@ -3,5 +3,4 @@ include:
   - services/auth/docker-compose.yaml
   - services/api/docker-compose.yaml
   - services/reverse-proxy/docker-compose.yaml
-  - specs/apis/docker-compose.yaml
-  - services/__template__/docker-compose.yaml
\ No newline at end of file
+  - specs/apis/docker-compose.yaml
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)